src/webextension/list/containers/item-filter.css

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.filter {
+  border: 1px solid #b6b6b6;
+  position: relative;
+  margin: 12px;
```

Comment:
Based on the specs, I think what we actually want here if for a new element (maybe a `<Toolbar/>`?) to wrap our `<ItemFilter/>` (put it in `src/webextension/list/manage/components/app.js`) and apply the margin there. That way, it'll be easier to move the "New Entry" button alongside the filter.